build.sh

```diff
@@ -4,6 +4,9 @@ set -e
 
 ORG_PATH="github.com/appc"
 REPO_PATH="${ORG_PATH}/docker2aci"
+# TODO: use "git describe --dirty" when we actually have tags pushed
+VERSION=$(git rev-parse --short HEAD)
+GLDFLAGS="-X github.com/appc/docker2aci/lib.Version=${VERSION}"
```

Comment:
Also, when vendoring docker2aci, the `Version` variable in `lib` package will not have the correct version (it will be `0.0.0+was-not-built-correctly` as hardcoded there), because we cannot expect project vendoring docker2aci code to pass a necessary linker flag with correct version.

A possible workaround could be hardcoding the version 0.9.0 and use some bump-version script, rkt style.

This would make docker2aci gogettable.